corehq/motech/repeaters/tests/test_repeater.py

```diff
@@ -1131,3 +1134,37 @@ def test_mess(self):
         })
         self.assertEqual(repeater.notify_addresses, ["admin@example.com",
                                                      "user@example.com"])
+
+
+class TestGetRetryInterval(SimpleTestCase):
+
+    def test_min_interval(self):
+        last_checked = fromisoformat("2020-01-01 00:00:00")
+        now = fromisoformat("2020-01-01 00:05:00")
+        interval = _get_retry_interval(last_checked, now)
+        self.assertEqual(interval, MIN_RETRY_WAIT)
+
+    def test_max_interval(self):
+        last_checked = fromisoformat("2020-01-01 00:00:00")
+        now = fromisoformat("2020-02-01 00:00:00")
+        interval = _get_retry_interval(last_checked, now)
+        self.assertEqual(interval, MAX_RETRY_WAIT)
+
+    def test_three_times_interval(self):
+        last_checked = fromisoformat("2020-01-01 00:00:00")
+        now = fromisoformat("2020-01-01 01:00:00")
+        interval = _get_retry_interval(last_checked, now)
+        self.assertEqual(interval, timedelta(hours=3))
+
+
+# TODO: When we upgrade to Python 3.7, use datetime.fromisoformat() instead
```

Comment:
you could also use `strptime` in the meantime

Reply:
:man_facepalming: Of course!